linode/manager
# refactor: [M3-6326] - MUI v5 - Components > DownloadCSV

## Description 📝
Refactor `DownloadCSV` component and re-use it throughout app.

## Major Changes 🔄
- Streamlining the use of this component since we always hide the CSV in order to use our Button component / style.
- Exporting the type `ButtonType` to be re-used
- This makes the CSV download buttons more prominent on `Linodes > Landing` and `Account > Maintenance`

## TODO:
- [x] Waiting on design/UX input on changes. Edit: Approved

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screen Shot 2023-05-05 at 10 53 43 AM](https://user-images.githubusercontent.com/125309814/236496720-8ce184f1-8dd4-43ab-87f3-b741a5b7cd32.png) | ![Screen Shot 2023-05-05 at 10 47 44 AM](https://user-images.githubusercontent.com/125309814/236496752-f954a327-2c0b-41b1-83dd-ca09729119cf.png) |
| ![Screen Shot 2023-05-05 at 10 50 38 AM](https://user-images.githubusercontent.com/125309814/236496809-b0b96749-72eb-45dd-a79f-fbb8b7aec4b9.png)| ![Screen Shot 2023-05-05 at 10 52 37 AM](https://user-images.githubusercontent.com/125309814/236496836-fad0737f-57d5-4e7f-8ea0-57a3b3726246.png)|

![Screen Shot 2023-05-05 at 10 47 25 AM](https://user-images.githubusercontent.com/125309814/236496993-126efc30-9aaa-41a8-a4f3-fdfb6cf5ae4b.png)

## How to test 🧪
1. Go to:
   - http://localhost:3000/account/billing/invoices/{ID}
   - http://localhost:3000/account/maintenance
   - http://localhost:3000/linodes
2. Observe changes and ensure CSV's still work
